certified-connectors/Impexium/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1850 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Impexium",
+    "description": "Impexium is a SaaS based AMS that allows businesses to automate  workflows, including adding activities, contacts, relationships and committee members.",
+    "version": "1.0",
+	"contact": {
+      "name": "Impexium Support",
+      "url": "support.impexium.com",
+      "email": "support@impexium.com"
+    }
+  },
+  "host": "automation.impexium.com",
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [],
+  "produces": [],
+  "paths": {
+    "/api/v1/Individuals": {
+      "post": {
+        "responses": {
+          "default": {
```

Comment:
@RjImpx Please 'replace' the "default" responses with expected status codes, otherwise users may see errors about incorrect schema instead of real error responses from your service in OpenAPI mode.